docs/pages/reference/backends.mdx

```diff
@@ -620,6 +620,182 @@ Config option priority is applied in the following order:
   are not supported in certain regions or environments or are only supported in GovCloud.
 </Admonition>
 
+## Athena
+
+If you are running Teleport on AWS, you can use
+[Athena](https://aws.amazon.com/athena/) based audit log system that uses
+[Parquet files](https://parquet.apache.org/) stored on
+[S3](https://aws.amazon.com/s3/) as a storage back-end to achieve High
+Availability. Athena backend supports only one type of Teleport data:
+
+- Audit log events
+
+The athena audit logs backend provides better scalability and searchability in
+comparison to DynamoDB.
+
+Athena audit logs are eventually consistent, this means that events won't be
+immediately displayed in UI. It can take up to 1 minute until they are
+visible (depending on `batchMaxInterval` setting and events load).
+
+### Infrastructure set up
+
+Teleport uses AWS SNS and SQS subscribed to topic, for publishing events.
+One of Auth instances read events in batches from SQS and converts them into
+Parquet format and sends to S3.
+During query Athena engine is use to search for events on S3, via Glue table
+which contains metadata.
+
+Infrastructure can be set up using following terraform script.
+
+<Details title="Terraform script" opened={false}>
+```
+Add here terraform examples when they are merged.
```

Comment:
I will add here valid file when https://github.com/gravitational/teleport/pull/29895 is merged, now it's making linter complaining.